integration/run_test.go

```diff
@@ -468,3 +468,178 @@ func TestKritisPods(t *testing.T) {
 		})
 	}
 }
+
+func TestKritisCron(t *testing.T) {
+	type testObject struct {
+		name string
+	}
+
+	type testRunCase struct {
+		description          string
+		dir                  string
+		args                 []string
+		deployments          []testObject
+		pods                 []testObject
+		deploymentValidation func(t *testing.T, d *appsv1.Deployment)
+		shouldSucceed        bool
+		shouldLabel          bool
+
+		remoteOnly bool
+		cleanup    func(t *testing.T)
+	}
+
+	var testCases = []testRunCase{
+		{
+			description: "nginx-no-digest-breakglass",
+			args: []string{"kubectl", "apply", "-f",
+				"integration/testdata/nginx/nginx-no-digest-breakglass.yaml"},
+			pods: []testObject{
+				{
+					name: "nginx-no-digest-breakglass",
+				},
+			},
+			shouldSucceed: true,
+			shouldLabel:   true,
+			dir:           "../",
+			cleanup: func(t *testing.T) {
+				cmd := exec.Command("kubectl", "delete", "-f",
+					"integration/testdata/nginx/nginx-no-digest-breakglass.yaml")
+				cmd.Dir = "../"
+				output, err := integration_util.RunCmdOut(cmd)
```

Comment:
instead of setting the Dir everywhere, we could say "../integration/testdata/nginx/nginx-no-digest-breakglass.yaml" in all the commands

Reply:
I will do this in another PR